centos/centos-7.x/centos-7.6-hpc/install_intel_libs.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
It will be good to verify that the current changes do not break 7.* images.

Reply:
Ok, I'll test the scripts on all 3 OS versions.

Reply:
Tested on CentOS 7.6, 7.7 and 8.1. All scripts and tests ran successfully.